-code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-40312/33386

ERROR: Build errors found during clang-tidy run.
```
SimCalorimetry/HcalSimAlgos/src/HcalSiPMHitResponse.cc:173:7: error: use of undeclared identifier 'elapsedTime' [clang-diagnostic-error]
      elapsedTime += dt;
      ^
Suppressed 823 warnings (822 in non-user code, 1 with check filters).
--
gmake: *** [config/SCRAM/GMake/Makefile.coderules:129: code-checks] Error 2
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
```

--- a/SimCalorimetry/HcalSimAlgos/src/HcalSiPMHitResponse.cc
+++ b/SimCalorimetry/HcalSimAlgos/src/HcalSiPMHitResponse.cc
@@ -170,7 +170,6 @@ void HcalSiPMHitResponse::addPEnoise(CLHEP::HepRandomEngine* engine) {
 
         sumnoisePE += noisepe;
       }
-      elapsedTime += dt;
 
     }  // precise time loop
 

--- a/SimG4CMS/FP420/plugins/FP420Test.cc
+++ b/SimG4CMS/FP420/plugins/FP420Test.cc
@@ -1412,7 +1412,7 @@ void FP420Test::update(const EndOfEvent* evt) {
             //UserNtuples->fillg42(float(intindex),1.);
             //UserNtuples->fillp208(float(intindex),float(nhit),1.);
             //UserNtuples->fillp211(float(intindex),float(themap1[index]),1.);
-            //totalEnergy += themap[index];
+            totalEnergy += themap[index];
           }  // for
         }    // for
         //UserNtuples->fillg39(nhitsecY,1.);

